CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
   false.
 - Fix a memory leak (freeing a malloc'd string - no a functional or security issue)
 - Title fixed when entering the 21st, 22nd and 23rd recovery word (was 21th, 22th, 23th) before.
+- Verifiable seed generation: when restoring from 24 recovery words, for the 24th word, show all 8 candidate words which result in a valid checksum.
+- Better error reporting on secure chip setup failures.
```

Comment:
Doesn't seem like touch-related... unless of course you fast-forward-merge :)

Reply:
Builds on https://github.com/digitalbitbox/bitbox02-firmware/pull/704, since I add a line below. Lazy way of avoiding a rebase conflict. Approve the other PR first? :)

Reply:
oh, sorry! I missed that one. :+1: